components/ReportList.vue

```diff
@@ -49,10 +51,24 @@ const getClasses = (dep) => {
     return "text-slate-500";
   }
 };
+
+const onSubmit = () => {
+  commit();
+  modalOpen.value = false;
+};
 </script>
 
 <template>
   <div v-if="suggestions.length" class="relative h-full">
+    <axo-modal
+      :open="modalOpen"
+      title="Commit changes"
+      submit-button-text="Commit"
+      @on-close="() => (modalOpen = false)"
+      @on-submit="onSubmit"
+      >This will make a commit on github with the changes you made here. Are you
```

Comment:
**suggestion, non-blocking:** This text would be nice in a variable, I think.

Reply:
done!